packages/ui/components/Icon/Icon.en.mdx

```diff
@@ -0,0 +1,131 @@
+import Row from '../Row'
+import Content from '../Content'
+import Br from '../Br'
+import TextInput from '../forms/TextInput'
+import Span from '../Span'
+import { Sandbox } from '@startupjs/docs'
+import {
+  faHouseUser,
+  faTable,
+  faShareAlt
+} from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-native-fontawesome'
+
+
+# Icon
+
+StartupJs uses `@fortawesome/react-native-fontawesome` library to implement icons. Icons are used to often compliment or substitute the meaning conveyed by words depending on the context.
+
+## Example
+
+```jsx
+import { Icon } from '@startupjs/ui'
+import {
+  faHouseUser,
+  faTable,
+  faShareAlt
+} from '@fortawesome/free-solid-svg-icons'
+```
+
+```jsx example
+return (
+  <Content>
+    <Row align='around'>
+      <Icon icon={faShareAlt} />
+      <Icon icon={faHouseUser} />
+      <Icon icon={faTable} />
+    </Row>
+  </Content>
+)
+```
+
+## Size
+
+Size of a icon can be modified using the `size` prop. Default `size` value is `m`. Pre defined icon size values are `xs`, `s`, `m`, `l`, `xl`, `xxl`. Size can also be arbitarily increased or decreased by supplying a number to `size` prop.
+
+```jsx
+import { Icon } from '@startupjs/ui'
+import {
+  faHouseUser,
+  faTable,
+  faShareAlt
+} from '@fortawesome/free-solid-svg-icons'
+```
+
+```jsx example
+const [numSize, setNumSize] = useState(80)
+return (
+  <Content>
+    <Row align='around'>
+      <Icon size='xs' icon={faShareAlt} />
+      <Icon size='s' icon={faHouseUser} />
+      <Icon icon={faTable} />
+      <Icon size='l' icon={faShareAlt} />
+      <Icon size='xl' icon={faHouseUser} />
+      <Icon size='xxl' icon={faTable} />
+    </Row>
+    <Br />
+    <TextInput
+      label='Type in number to change icon size'
+      value={numSize}
+      onChangeText={newVal => setNumSize(Number(newVal))}
+    />
+    <Br />
+    <Icon size={numSize} icon={faTable} />
+  </Content>
+)
+```
+
+## Color
+
+Icon color can be customized by providing hex string to the `color` prop.
+
+```jsx
+import { Icon } from '@startupjs/ui'
+import {
+  faHouseUser,
+  faTable,
+  faShareAlt
+} from '@fortawesome/free-solid-svg-icons'
+```
+
+```jsx example
+return (
+  <Content>
+    <Row align='around'>
+      <Icon color='#24B47E' icon={faShareAlt} />
+      <Icon color='#6772E5' icon={faHouseUser} />
+      <Icon color='#F5BE59' icon={faTable} />
+    </Row>
+  </Content>
+)
+```
+
+## Custom Icon
+
+For custom icon the `icon` prop can be supplied with a function which returns a custom icon. The `Icon` component invokes the function with the appropriate props.
+
+```jsx
+import { Icon } from '@startupjs/ui'
+import { FontAwesomeIcon } from '@fortawesome/react-native-fontawesome'
+import { faShareAlt } from '@fortawesome/free-solid-svg-icons'
+```
+
+```jsx example
+return (
+  <Content>
+    <Icon
+      size={40}
+      color='#F5BE59'
+      icon={(props) => (<FontAwesomeIcon {...props} icon={faShareAlt} />)} 
```

Comment:
Example here should be on how to use the custom SVG file for an icon, therefore it also makes sense to change the heading and description of this section from `Custom Icon` to `Custom SVG file`.

And the way to use it, I think, is just to import the file and pass it in:

```jsx
import BirdSvg from './bird.svg'

<Icon
  color='#F5BE59'
  icon=BirdSvg
/>
```